common/G4_hFarFwdBeamLine_EIC.C

```diff
@@ -12,10 +12,14 @@
 #include <eicg4zdc/EICG4ZDCNtuple.h>
 #include <eicg4zdc/EICG4ZDCSubsystem.h>
 
+#include <g4eval/FarForwardEvaluator.h>
```

Comment:
Please first add this Eval to EIC daily build at https://github.com/eic/fun4all_eicdetectors/tree/master/analysis/eicevaluator

Reply:
Thanks Jing, I will add the forward evaluator to  https://github.com/eic/fun4all_eicdetectors/tree/master/analysis/eicevaluator